cmd/list.go

```diff
@@ -82,21 +81,8 @@ func newListCmd() *cobra.Command {
 		Short: "List all workflows and sessions.",
 		Long:  listDesc,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if o.token == "" {
-				o.token = os.Getenv("REANA_ACCESS_TOKEN")
-			}
-			o.serverURL = os.Getenv("REANA_SERVER_URL")
-
-			if err := validation.ValidateAccessToken(o.token); err != nil {
-				return err
-			}
-			if err := validation.ValidateServerURL(o.serverURL); err != nil {
-				return err
-			}
-			if err := o.run(cmd); err != nil {
-				return err
-			}
-			return nil
+			o.serverURL = viper.GetString("server-url")
```

Comment:
On an unrelated note, it might be a good idea to also have a command flag for the server URL in the future. Do you know if there is a reason why we do not do it in the Python client?

Reply:
Indeed I had the same question why there is a flag for a token and not for a server url. @tiborsimko, do you remember the reason?